src/org/httpkit/client.clj

```diff
@@ -193,7 +195,9 @@
                                 ;; only the 4 support now
                                 (case as :auto 1 :text 2 :stream 3 :byte-array 4))
         effective-proxy-url (if proxy-host (str proxy-host ":" proxy-port) proxy-url)
-        cfg (RequestConfig. method headers body timeout keepalive effective-proxy-url tunnel?)]
+        conn-timeout (if timeout timeout conn-timeout)
+        read-timeout (if timeout timeout read-timeout)
```

Comment:
Same comment as above (though just skimmed this so may be missing something) :-)